src/unity/python/turicreate/test/test_audio_functionality.py

```diff
@@ -11,6 +11,7 @@
 
 from .util import TempDirectory
 
+from copy import copy
```

Comment:
I've wondered this earlier looking at our test code as well -- why don't we hide imports in the test code?

Reply:
We hide imports in non test code so users know which methods/classes we provide. Users aren't typically going to import unit test files. So we don't worry about it there.